examples/nndvi_example.py

```diff
@@ -0,0 +1,30 @@
+"""
+NNDVI Example -- currently used to debug false positives issue #36.
```

Comment:
Should remove reference to this issue number, and fold this into the other drift examples notebook.

If it takes an especially long time to run, then it would be worth noting that in the heading, like we note for LFR.